src/llamafactory/extras/constants.py

```diff
@@ -66,6 +66,21 @@
 
 SUPPORTED_CLASS_FOR_S2ATTN = {"llama"}
 
+SUPPORTED_CLASS_FOR_MULTIPACK = [
```

Comment:
is it "efficient_packing" rather than "multipack"?

Reply:
yes, I just fixed.